notebooks/BoostHistogramHandsOn.ipynb

```diff
@@ -450,21 +450,14 @@
    "outputs": [],
    "source": [
     "def plothist2d(h):\n",
-    "    X, Y = h.axes.edges\n",
-    "    return plt.pcolormesh(X.T, Y.T, h.view().T)"
+    "    return plt.pcolormesh(*h.axes.edges.T, h.view().T)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This is transposed because it is ij indexed instead of xy. You can also do:\n",
-    "\n",
-    "```python\n",
-    "X, Y = h.axes.edges\n",
-    "X, Y = np.broadcast_arrays(X, Y)\n",
-    "return plt.pcolormesh(X, Y, h)\n",
-    "```"
+    "This is transposed because pcolormesh requires ij indexing instead of xy."
```

Comment:
This is not a clear explanation. Better just to forward to the pcolormesh docs, https://matplotlib.org/3.2.2/api/_as_gen/matplotlib.pyplot.pcolormesh.html#axes-pcolormesh-grid-orientation

Reply:
It doesn't help that it's wrong, either, I had ij and xy backwards. I'll just point at the docs...